enterprise/backend/src/metabase_enterprise/content_translation/routes.clj

```diff
@@ -0,0 +1,68 @@
+(ns metabase-enterprise.content-translation.routes
+  "Endpoints relating to the translation of user-generated content"
+  (:require
+   [clojure.data.csv :as csv]
+   [clojure.java.io :as io]
+   [metabase-enterprise.api.routes.common :as ee.api.common]
+   [metabase-enterprise.content-translation.dictionary :as dictionary]
+   [metabase.api.common :as api]
+   [metabase.api.macros :as api.macros]
+   [metabase.content-translation.models :as ct]
+   [metabase.util.i18n :refer [deferred-tru tru]]
+   [metabase.util.malli.schema :as ms]))
+
+(set! *warn-on-reflection* true)
+
+(def ^:private http-status-content-too-large 413)
+
+;; The maximum size of a content translation dictionary is 1.5MiB
+;; This should equal the maxContentDictionarySizeInMiB variable in the frontend
+(def ^:private max-content-translation-dictionary-size (* 1.5 1024 1024))
```

Comment:
I always like including units in var names when it's potentially relevant, ie `max-content-translation-dictionary-size-bytes`.

You could also do something like

```
(def ^:private max-content-translation-dictionary-size-mb 1.5)
(def ^:private max-content-translation-dictionary-size-bytes (* max-content-translation-dictionary-size-mb 1024 1024))
```

then later when you're throwing the too-big exception you don't need to worry about dividing by `(* 1024 1024)`. Not blocking though.